modules/topicsFpdModule.js

```diff
@@ -58,10 +64,31 @@ export function getTopics(doc = document) {
   return topics;
 }
 
+// function to fetch the cached topic data from storage for bidders and return it
+export function getTpsForBidderFromStorage() {
+  let cachedTopicData = [];
+  const topics = config.getConfig('userSync.topics');
+  let biddersList = Object.keys(topics.bidders || []);
+  Object.keys(window.localStorage).forEach((storeItem) => {
```

Comment:
With the strategy you are using here, I would store a single map where the key is  `` `${segment.name}:${segment.ext.segclass}` `` (because I believe `segment.name` would be an alias of some kind for the bidder? otherwise some other combination that would uniquely identify the segment).